build/release/README.md

```diff
@@ -79,6 +79,12 @@ After the PR is merged, you can tag the release with the beta tag (`v1.13.0-beta
 
 **IMPORTANT** Before tagging the release, open a new PR to update the documentation and example manifest tags to the release version.
 
+The script  `set-release-ver.sh` can help preparing these changes:
+
+```console
+# Pass the new version
```

Comment:
I think "pass the new version" is better left as an instruction above. It'll be easily copy-pasted with the line below as part of the snippet. Seems best to leave it so it won't risk causing any issues when that happens

Reply:
done